package.json

```diff
@@ -434,4 +434,4 @@
     "node": "10.15.2",
     "yarn": "^1.10.1"
   }
-}
+}
```

Comment:
new line here. That's weird your IDE didn't auto-add it.

Reply:
Somehow it allowed a new line for all the files but this one (It was not only not adding it, but removing it if I added) lol. Thanks for notice, just upgrade my VSCode and added a rule for it :D